ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml

```diff
@@ -93,11 +93,7 @@ EditFeatureAttachmentsSample {
                     margins: 10
                 }
 
-                text: "Attachments"
-                font {
-                    bold: true
-                    pointSize: 20
-                }
+                text: "Attachments"; font {bold: true; pixelSize: 20;}
```

Comment:
Formatting seems a bit odd in a few places. Do we want to revert back to these or no?